packages/sanes-chrome-extension/scripts/remove_key.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Why deleting this key? As far as I remember it was created in a deterministic way to be unique. Why not use it for uploading it to the chrome extension?

Reply:
the `key` should not be used at all:

> You don't usually need to use this value. Instead, write your code so that the key value doesn't matter by using relative paths and extension.getURL. 

https://developer.chrome.com/apps/manifest/key

Foir development it is conveneient to ensure you always get the same ID. When uploading the extension to the web store, the `key` property is forbidden. If it is set, the upload is rejected. The extension ID is assigned by the web store.
```

Reply:
Perfect. 👌